group/curve25519/proj.go

```diff
@@ -238,6 +248,11 @@ func (P *projPoint) Mul(s kyber.Scalar, G kyber.Point) kyber.Point {
 	return P
 }
 
+// This implementation only supports variable-time operations
+func (P *projPoint) SetVarTime(varTime bool) bool {
+	return true
+}
```

Comment:
I think so yes. I was suggesting the idea of panic before but no clear consensus have emerged. I'm up for it.